master/buildbot/process/properties.py

```diff
@@ -603,6 +613,17 @@ def _parse_src(arg):
             codebase = attr = repl = None
         return _SourceStampDict(codebase), attr, repl
 
+    def _parse_worker(self, arg):
+        try:
+            prop, repl = arg.split(":", 1)
+        except ValueError:
+            prop, repl = arg, None
+        if not Interpolate.identifier_re.match(prop):
+            config.error(
```

Comment:
I think we need a test that covers this error.

Reply:
asch, was supposed to remove the check altogether